docs/changelog.md

```diff
@@ -1,3 +1,10 @@
+## 0.20.3 (Jul 22, 2021)
+
+### Add
+
+*  Add `shared_history` param to `Bot.create_chat` method.
```

Comment:
Add possibility to create chats with enabled `shared_history` option (`Bot.create_chat`).

Reply:
Done